assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/04-deployment.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: admission-webhook-server
+  labels:
+    name: admission-webhook-server
```

Usually `app` or `k8s-app` labels are being used.
```suggestion
    k8s-app: admission-webhook-server
```